src/main/java/io/temporal/internal/testservice/StateMachines.java

```diff
@@ -1858,4 +1845,23 @@ private static void failExternalCancellation(
             .build();
     ctx.addEvent(event);
   }
+
+  // Mimics the default activity retry policy of a standard Temporal server.
+  static RetryPolicy ensureDefaultFieldsForActivityRetryPolicy(RetryPolicy originalPolicy) {
+    return RetryPolicy.newBuilder()
+        .setInitialIntervalInSeconds(
+            originalPolicy.getInitialIntervalInSeconds() == 0
+                ? 1
```

Comment:
We already have constants for some defaults in this file. I would convert these to constants as well.

Reply:
good point. added statics at the top of the file